sample_code/v25/circuitpython/README.md

```diff
@@ -0,0 +1,47 @@
+## Getting Started
+
+### Update Bootloader to version 6.3
+- Plugin your device via USB to your comptuer. The storage drive will show up as an external USB disk.
+- Double tap reset button to get to bootloader, the device will un-mount the disk and re-mount as the bootloader ("ITSYBTSY")
+- Download the lastest bootloader https://circuitpython.org/board/itsybitsy_m0_express/
+- Extract, copy the UF2 file to the ITSYBTSY drive that shows up on your PC. No need to rename file. Device will reboot immediately after copy.
+- The device will reboot and mount its storage disk. Probably called "NO_NAME" however you can rename it to whatever you please.
+- Open the `boot_out.txt` file. You should see updated firmware version and date.
+
+### Update lib packages
+- Download the latest 6.3.0 CircuitPython here: https://github.com/adafruit/Adafruit_CircuitPython_Bundle/releases/download/20210611/adafruit-circuitpython-bundle-6.x-mpy-20210611.zip
+- Extract and open the `lib/` folder, find and copy these files over to your device's `lib/` folder:
+  - `adafruit_hid/` (folder)
+  - `adafruit_dotstar.mpy`
+  - `adafruit_matrixkeypad.mpy`
+  - `digitalio.mpy`
```

Comment:
I followed these instructions, however `digitalio.mpy` does not exist

Reply:
It _does_ exist in the `adafruit_seesaw` and `adafruit_esp32spi` folders. Should one of those be copied instead? 

```
$pwd
/Users/spencerowen/Downloads/adafruit-circuitpython-bundle-6.x-mpy-20210611/lib
find . -type f -name digitalio.mpy
./adafruit_seesaw/digitalio.mpy
./adafruit_esp32spi/digitalio.mpy
```